extension/js/common/composer.ts

```diff
@@ -1919,7 +1919,14 @@ export class Composer {
       this.S.cached('icon_show_prev_msg').show().addClass('progress');
       Xss.sanitizeAppend(this.S.cached('icon_show_prev_msg'), '<div id="loader">0%</div>');
       this.resizeComposeBox();
-      this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      try {
+        this.messageToReplyOrForward = await this.getAndDecryptPreviousMessage((progress) => this.setQuoteLoaderProgress(progress + '%'));
+      } catch (e) {
+        Catch.reportErr(e);
```

Comment:
Here you can do:
```js
if(Api.err.isSignificant(e)) {
  Catch.reportErr(e);
}
```
That way it will only report errors that are not server 5xx, network errors and similar (eg stuff that we should actually fix).

Reply:
Yeah, I forgot to add this. Thanks!